nilearn/input_data/nifti_maps_masker.py

```diff
@@ -292,18 +246,28 @@ def transform(self, imgs, confounds=None):
                         target_shape=ref_img.shape[:3],
                         target_affine=ref_img.get_affine())
 
+        def extraction_function(imgs):
```

Comment:
Wow can you get away with that, without a self and without having it a staticmethod or classmethod? I thought the first argument had to be an instance of the class from the method ...

Also can you use a consistent name. You used extraction_function here and extraction_func somewhere else.